testdata/ansible/memcached-operator/requirements.yml

```diff
@@ -1,10 +1,10 @@
 ---
 collections:
   - name: community.kubernetes
-    version: "1.2.1"
+    version: "2.0.1"
   - name: operator_sdk.util
     version: "0.4.0"
   - name: kubernetes.core
-    version: "2.2.0"
+    version: "2.3.1"
   - name: cloud.common
-    version: "2.1.0"
+    version: "2.1.1"
```

Comment:
@camilamacedo86 The version doesn't match with the docs. 

![image](https://user-images.githubusercontent.com/7982347/176166669-64092afc-78b7-4480-8180-0a523db3ee22.png)